### PR TITLE
Pioneer 13 deployment and recovery updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updated deployment and recovery info for all profiler moorings on Pioneer 13 cruise:
deployment and recovery dates/times
recovery cruise ID
lat/lon
water depths and deployment depths (It looks like earlier deployments need to have deployment depths updated per Sheri's email from Sept 17.  I did NOT update those on this pull request).

### CODE REVIEW

File: deployment/CP01CNPM_Deploy.csv [hunk 1]
```diff
@@ -17,12 +17,12 @@ AR-31,,AR-34,,CP01CNPM-WFP01-02-DOFSTK000,2,1,2018-11-02T14:40:00,2019-04-04T12:
...
+AR-39,,,,CP01CNPM-SBS01-00-STCENG000,3,1,2019-10-07T13:44:00,,CGMCP-01CNPM-00003,,CGCON-BSTC01-00013,40.139158,-70.779413,,0,132,
+AR-39,,,,CP01CNPM-SBS01-01-MOPAK0000,3,1,2019-10-07T13:44:00,,CGMCP-01CNPM-00003,,CGINS-MOPAK0-11360,40.139158,-70.779413,,0,132,
+AR-39,,,,CP01CNPM-WFP01-00-WFPENG000,3,1,2019-10-07T13:44:00,,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGCON-CWFPCP-30002,40.139158,-70.779413,,111,132,
+AR-39,,,,CP01CNPM-WFP01-01-VEL3DK000,3,1,2019-10-07T13:44:00,,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-VEL3DK-00470,40.139158,-70.779413,,111,132,
+AR-39,,,,CP01CNPM-WFP01-02-DOFSTK000,3,1,2019-10-07T13:44:00,,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-DOFSTK-03523,40.139158,-70.779413,,111,132,
+AR-39,,,,CP01CNPM-WFP01-03-CTDPFK000,3,1,2019-10-07T13:44:00,,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,40.139158,-70.779413,,111,132,
+AR-39,,,,CP01CNPM-WFP01-04-FLORTK000,3,1,2019-10-07T13:44:00,,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,40.139158,-70.779413,,111,132,
+AR-39,,,,CP01CNPM-WFP01-05-PARADK000,3,1,2019-10-07T13:44:00,,CGMCP-01CNPM-00003,CGVEH-CWFPCP-30002,CGINS-PARADK-20542,40.139158,-70.779413,,111,132,
```
Comment (reedan88) [inline deployment/CP01CNPM_Deploy.csv]: Looks good.

File: deployment/CP02PMCI_Deploy.csv [hunk 2]
```diff
@@ -98,21 +98,21 @@ AR-31,,AR-34,,CP02PMCI-WFP01-02-DOFSTK000,11,1,2018-11-05T14:53:00,2019-04-18T21
...
+AR-39,,,,CP02PMCI-SBS01-00-STCENG000,13,1,2019-10-07T18:23:00,,CGMCP-02PMCI-00013,,CGCON-BSTC01-00017,40.226972,-70.877882,,0,126,
+AR-39,,,,CP02PMCI-SBS01-01-MOPAK0000,13,1,2019-10-07T18:23:00,,CGMCP-02PMCI-00013,,CGINS-MOPAK0-58478,40.226972,-70.877882,,0,126,
+AR-39,,,,CP02PMCI-WFP01-00-WFPENG000,13,1,2019-10-07T18:23:00,,CGMCP-02PMCI-00013,CGVEH-CWFPCP-99101,CGCON-CWFPCP-99101,40.226972,-70.877882,,103,126,
+AR-39,,,,CP02PMCI-WFP01-01-VEL3DK000,13,1,2019-10-07T18:23:00,,CGMCP-02PMCI-00013,CGVEH-CWFPCP-99101,CGINS-VEL3DK-00015,40.226972,-70.877882,,103,126,
+AR-39,,,,CP02PMCI-WFP01-02-DOFSTK000,13,1,2019-10-07T18:23:00,,CGMCP-02PMCI-00013,CGVEH-CWFPCP-99101,CGINS-DOFSTK-02978,40.226972,-70.877882,,103,126,
+AR-39,,,,CP02PMCI-WFP01-03-CTDPFK000,13,1,2019-10-07T18:23:00,,CGMCP-02PMCI-00013,CGVEH-CWFPCP-99101,CGINS-CTDPFK-00111,40.226972,-70.877882,,103,126,
+AR-39,,,,CP02PMCI-WFP01-04-FLORTK000,13,1,2019-10-07T18:23:00,,CGMCP-02PMCI-00013,CGVEH-CWFPCP-99101,CGINS-FLORTK-01029,40.226972,-70.877882,,103,126,
+AR-39,,,,CP02PMCI-WFP01-05-PARADK000,13,1,2019-10-07T18:23:00,,CGMCP-02PMCI-00013,CGVEH-CWFPCP-99101,CGINS-PARADK-20437,40.226972,-70.877882,,103,126,
```
Comment (reedan88) [inline deployment/CP02PMCI_Deploy.csv]: Looks good.

File: deployment/CP02PMCO_Deploy.csv [hunk 3]
```diff
@@ -89,21 +89,21 @@ AR-31,,AR-34,,CP02PMCO-WFP01-02-DOFSTK000,11,1,2018-11-02T20:28:00,2019-04-23T16
...
+AR-39,,,,CP02PMCO-SBS01-00-STCENG000,13,1,2019-10-08T02:02:00,,CGMCP-02PMCO-00013,,CGCON-BSTC01-00016,40.096798,-70.879063,,0,146,
+AR-39,,,,CP02PMCO-SBS01-01-MOPAK0000,13,1,2019-10-08T02:02:00,,CGMCP-02PMCO-00013,,CGINS-MOPAK0-43263,40.096798,-70.879063,,0,146,
+AR-39,,,,CP02PMCO-WFP01-00-WFPENG000,13,1,2019-10-08T02:02:00,,CGMCP-02PMCO-00013,CGVEH-CWFPCP-99102,CGCON-CWFPCP-99102,40.096798,-70.879063,,125,146,
+AR-39,,,,CP02PMCO-WFP01-01-VEL3DK000,13,1,2019-10-08T02:02:00,,CGMCP-02PMCO-00013,CGVEH-CWFPCP-99102,CGINS-VEL3DK-00013,40.096798,-70.879063,,125,146,
+AR-39,,,,CP02PMCO-WFP01-02-DOFSTK000,13,1,2019-10-08T02:02:00,,CGMCP-02PMCO-00013,CGVEH-CWFPCP-99102,CGINS-DOFSTK-02499,40.096798,-70.879063,,125,146,
+AR-39,,,,CP02PMCO-WFP01-03-CTDPFK000,13,1,2019-10-08T02:02:00,,CGMCP-02PMCO-00013,CGVEH-CWFPCP-99102,CGINS-CTDPFK-00113,40.096798,-70.879063,,125,146,
+AR-39,,,,CP02PMCO-WFP01-04-FLORTK000,13,1,2019-10-08T02:02:00,,CGMCP-02PMCO-00013,CGVEH-CWFPCP-99102,CGINS-FLORTK-01031,40.096798,-70.879063,,125,146,
+AR-39,,,,CP02PMCO-WFP01-05-PARADK000,13,1,2019-10-08T02:02:00,,CGMCP-02PMCO-00013,CGVEH-CWFPCP-99102,CGINS-PARADK-20447,40.096798,-70.879063,,125,146,
```
Comment (reedan88) [inline deployment/CP02PMCO_Deploy.csv]: Looks good.

File: deployment/CP02PMUI_Deploy.csv [hunk 4]
```diff
@@ -98,21 +98,21 @@ AR-31,,AR-34,,CP02PMUI-WFP01-02-DOFSTK000,12,1,2018-11-04T16:43:00,2019-04-19T00
...
+AR-39,,,,CP02PMUI-SBS01-00-STCENG000,14,1,2019-10-06T18:26:00,,CGMCP-02PMUI-00014,,CGCON-BSTC01-00042,40.365165,-70.769673,,0,93,
+AR-39,,,,CP02PMUI-SBS01-01-MOPAK0000,14,1,2019-10-06T18:26:00,,CGMCP-02PMUI-00014,,CGINS-MOPAK0-43252,40.365165,-70.769673,,0,93,
+AR-39,,,,CP02PMUI-WFP01-00-WFPENG000,14,1,2019-10-06T18:26:00,,CGMCP-02PMUI-00014,CGVEH-CWFPCP-71401,CGCON-CWFPCP-71401,40.365165,-70.769673,,69,93,
+AR-39,,,,CP02PMUI-WFP01-01-VEL3DK000,14,1,2019-10-06T18:26:00,,CGMCP-02PMUI-00014,CGVEH-CWFPCP-71401,CGINS-VEL3DK-00012,40.365165,-70.769673,,69,93,
+AR-39,,,,CP02PMUI-WFP01-02-DOFSTK000,14,1,2019-10-06T18:26:00,,CGMCP-02PMUI-00014,CGVEH-CWFPCP-71401,CGINS-DOFSTK-00209,40.365165,-70.769673,,69,93,
+AR-39,,,,CP02PMUI-WFP01-03-CTDPFK000,14,1,2019-10-06T18:26:00,,CGMCP-02PMUI-00014,CGVEH-CWFPCP-71401,CGINS-CTDPFK-00093,40.365165,-70.769673,,69,93,
+AR-39,,,,CP02PMUI-WFP01-04-FLORTK000,14,1,2019-10-06T18:26:00,,CGMCP-02PMUI-00014,CGVEH-CWFPCP-71401,CGINS-FLORTK-00830,40.365165,-70.769673,,69,93,
+AR-39,,,,CP02PMUI-WFP01-05-PARADK000,14,1,2019-10-06T18:26:00,,CGMCP-02PMUI-00014,CGVEH-CWFPCP-71401,CGINS-PARADK-20525,40.365165,-70.769673,,69,93,
```
Comment (reedan88) [inline deployment/CP02PMUI_Deploy.csv]: Looks good.

File: deployment/CP02PMUO_Deploy.csv [hunk 5]
```diff
@@ -107,21 +107,21 @@ AR-31,,AR-34,,CP02PMUO-WFP01-02-DOFSTK000,12,1,2018-11-07T21:08:00,2019-04-22T20
...
+AR-39,,,,CP02PMUO-SBS01-00-STCENG000,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,,CGCON-BSTC01-00005,39.941912,-70.779573,,0,449,
+AR-39,,,,CP02PMUO-SBS01-01-MOPAK0000,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,,CGINS-MOPAK0-11358,39.941912,-70.779573,,0,449,
+AR-39,,,,CP02PMUO-WFP01-00-WFPENG000,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,CGVEH-CWFPCP-10301,CGCON-CWFPCP-10301,39.941912,-70.779573,,425,449,
+AR-39,,,,CP02PMUO-WFP01-01-VEL3DK000,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,CGVEH-CWFPCP-10301,CGINS-VEL3DK-00035,39.941912,-70.779573,,425,449,
+AR-39,,,,CP02PMUO-WFP01-02-DOFSTK000,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,CGVEH-CWFPCP-10301,CGINS-DOFSTK-02708,39.941912,-70.779573,,425,449,
+AR-39,,,,CP02PMUO-WFP01-03-CTDPFK000,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,CGVEH-CWFPCP-10301,CGINS-CTDPFK-00124,39.941912,-70.779573,,425,449,
+AR-39,,,,CP02PMUO-WFP01-04-FLORTK000,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,CGVEH-CWFPCP-10301,CGINS-FLORTK-01117,39.941912,-70.779573,,425,449,
+AR-39,,,,CP02PMUO-WFP01-05-PARADK000,14,1,2019-10-14T18:09:00,,CGMCP-02PMUO-00014,CGVEH-CWFPCP-10301,CGINS-PARADK-20464,39.941912,-70.779573,,425,449,
```
Comment (reedan88) [inline deployment/CP02PMUO_Deploy.csv]: Looks good.

File: deployment/CP03ISPM_Deploy.csv [hunk 6]
```diff
@@ -17,12 +17,12 @@ AR-31,,AR-34,,CP03ISPM-WFP01-02-DOFSTK000,2,1,2018-10-23T02:55:00,2019-04-17T11:
...
+AR-39,,,,CP03ISPM-SBS01-00-STCENG000,3,1,2019-10-06T13:41:00,,CGMCP-03ISPM-00003,,CGCON-BSTC01-00018,40.364817,-70.888668,,0,92,
+AR-39,,,,CP03ISPM-SBS01-01-MOPAK0000,3,1,2019-10-06T13:41:00,,CGMCP-03ISPM-00003,,CGINS-MOPAK0-58481,40.364817,-70.888668,,0,92,
+AR-39,,,,CP03ISPM-WFP01-00-WFPENG000,3,1,2019-10-06T13:41:00,,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGCON-CWFPCP-30001,40.364817,-70.888668,,68,92,
+AR-39,,,,CP03ISPM-WFP01-01-VEL3DK000,3,1,2019-10-06T13:41:00,,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-VEL3DK-00468,40.364817,-70.888668,,68,92,
+AR-39,,,,CP03ISPM-WFP01-02-DOFSTK000,3,1,2019-10-06T13:41:00,,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-DOFSTK-03506,40.364817,-70.888668,,68,92,
+AR-39,,,,CP03ISPM-WFP01-03-CTDPFK000,3,1,2019-10-06T13:41:00,,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,40.364817,-70.888668,,68,92,
+AR-39,,,,CP03ISPM-WFP01-04-FLORTK000,3,1,2019-10-06T13:41:00,,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,40.364817,-70.888668,,68,92,
+AR-39,,,,CP03ISPM-WFP01-05-PARADK000,3,1,2019-10-06T13:41:00,,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-PARADK-20541,40.364817,-70.888668,,68,92,
```
Comment (reedan88) [inline deployment/CP03ISPM_Deploy.csv]: Looks good.

File: deployment/CP04OSPM_Deploy.csv [hunk 7]
```diff
@@ -82,21 +82,21 @@ AR-31,,AR-34,,CP04OSPM-WFP01-02-DOFSTK000,10,1,2018-11-06T14:36:00,2019-04-22T16
...
+AR-39,,,,CP04OSPM-SBS01-00-STCENG000,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,,CGCON-BSTC01-00012,39.935558,-70.880328,,0,451,
+AR-39,,,,CP04OSPM-SBS11-02-MOPAK0000,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,,CGINS-MOPAK0-11347,39.935558,-70.880328,,0,451,
+AR-39,,,,CP04OSPM-WFP01-00-WFPENG000,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGCON-CWFPCP-99105,39.935558,-70.880328,,405,451,
+AR-39,,,,CP04OSPM-WFP01-01-VEL3DK000,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-VEL3DK-00212,39.935558,-70.880328,,405,451,
+AR-39,,,,CP04OSPM-WFP01-02-DOFSTK000,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-DOFSTK-03370,39.935558,-70.880328,,405,451,
+AR-39,,,,CP04OSPM-WFP01-03-CTDPFK000,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-CTDPFK-00169,39.935558,-70.880328,,405,451,
+AR-39,,,,CP04OSPM-WFP01-04-FLORTK000,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-FLORTK-01466,39.935558,-70.880328,,405,451,
+AR-39,,,,CP04OSPM-WFP01-05-PARADK000,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,CGVEH-CWFPCP-99105,CGINS-PARADK-20513,39.935558,-70.880328,,405,451,
```
Comment (reedan88) [inline deployment/CP04OSPM_Deploy.csv]: Looks good.

File: deployment/CP01CNPM_Deploy.csv (hunk 1 above)
Comment (snwhite) [reply deployment/CP01CNPM_Deploy.csv]: Good

File: deployment/CP02PMCI_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/CP02PMCI_Deploy.csv]: Good.

File: deployment/CP02PMCO_Deploy.csv (hunk 3 above)
Comment (snwhite) [reply deployment/CP02PMCO_Deploy.csv]: Good.

File: deployment/CP02PMUI_Deploy.csv (hunk 4 above)
Comment (snwhite) [reply deployment/CP02PMUI_Deploy.csv]: Good.

File: deployment/CP02PMUO_Deploy.csv (hunk 5 above)
Comment (snwhite) [reply deployment/CP02PMUO_Deploy.csv]: Good.

File: deployment/CP03ISPM_Deploy.csv (hunk 6 above)
Comment (snwhite) [reply deployment/CP03ISPM_Deploy.csv]: Good.

File: deployment/CP04OSPM_Deploy.csv (hunk 7 above)
Comment (snwhite) [reply deployment/CP04OSPM_Deploy.csv]: Good.

**Comment by rtravis-whoi** [comment]
> 
> 
> All looks good.
> In the future, when we are "touching" a mooring to add recovery information, let's also correct depths for that mooring.

Sounds good - I figured I'd tackle all of them at a later time rather than bog this PR down with changes to most or all of the previous deployments for each mooring.

**Pull request merged**